parser_test.go

```diff
@@ -17,13 +17,21 @@ func runParserWith(line string) *Parser {
 }
 
 func TestParserWithMatchingInputGo16(t *testing.T) {
-	line := "gc #88 @3.243s 9%: 0.040+16+1.0+5.9+0.34 ms clock, 0.16+16+0+18/5.7/11+1.3 ms cpu, 32->33->19 MB, 33 MB goal, 4 P"
+	line := "gc 763 @77536.239s 1%: 0.11+2192+0.75 ms clock, 0.92+9269/4379/3243+6.0 ms cpu, 6370->6390->3298 MB, 6533 MB goal, 8 P"
```

Comment:
Thanks for reminding me, I had forgotten the details. And that string does
change, a lot.

On Tue, 15 Mar 2016, 18:58 Vincent Bonmalais, <notifications@github.com>
wrote:

> In parser_test.go
> <https://github.com/davecheney/gcvis/pull/26#discussion_r56125958>:
>
> > @@ -17,13 +17,21 @@ func runParserWith(line string) *Parser {
> >  }
> >
> >  func TestParserWithMatchingInputGo16(t *testing.T) {
> > -	line := "gc #88 @3.243s 9%: 0.040+16+1.0+5.9+0.34 ms clock, 0.16+16+0+18/5.7/11+1.3 ms cpu, 32->33->19 MB, 33 MB goal, 4 P"
> > +	line := "gc 763 @77536.239s 1%: 0.11+2192+0.75 ms clock, 0.92+9269/4379/3243+6.0 ms cpu, 6370->6390->3298 MB, 6533 MB goal, 8 P"
>
> It is the test for 1.6 but at the time, 1.6 was not released yet. As far
> as I tested, the format was close enough to not warrant a new regexp [image:
> :smile:].
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/davecheney/gcvis/pull/26/files#r56125958
>